features/admin.alternative-login-identifier.v1/pages/alternative-login-identifier-page.tsx

```diff
@@ -332,27 +342,25 @@ const AlternativeLoginIdentifierInterface: FunctionComponent<AlternativeLoginIde
             });
     };
 
-    // Define a function to update claim properties with `isUnique` property.
-    const updateClaimProperties =(claim: Claim, checkedClaims: string[]) => {
-        let isClaimUpdate: boolean = false;
-        let updatedClaimProperties: Property[] = [ ...claim.properties ];
-        const isUniqueIndex: number = claim?.properties?.findIndex((property: Property) => property.key === "isUnique");
-
-        if (checkedClaims?.includes(claim.claimURI)) {
-            if (isUniqueIndex !== -1 && claim.properties[isUniqueIndex].value === "false") {
-                isClaimUpdate = true;
-                updatedClaimProperties[isUniqueIndex].value = "true";
-            } else if (isUniqueIndex === -1) {
-                isClaimUpdate = true;
-                updatedClaimProperties.push({ key: "isUnique", value: "true" });
-            }
-        } else if (isUniqueIndex !== -1) {
-            isClaimUpdate = true;
-            updatedClaimProperties = updatedClaimProperties.filter((property: Property) =>
-                property.key !== "isUnique");
-        }
-
-        return { isClaimUpdate, updatedClaimProperties };
+    /**
+     * Updates the uniqueness scope of a claim if it's selected and needs to be updated to ACROSS_USERSTORES.
+     * Does not modify claims that are not selected.
+     */
+    const updateClaimUniquenessScope = (claim: Claim, checkedClaims: string[]) => {
+        const isSelected: boolean = checkedClaims?.includes(claim.claimURI);
+
+        const needsUpdate: boolean = isSelected &&
```

Comment:
More descriptive variable name.
```suggestion
        const shouldUpdateClaim: boolean = isSelected &&
```

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7106/commits/3088d327c477257341387a57516db4d9929fbfaa